renovate.json

```diff
@@ -73,14 +79,13 @@
     },
     {
       "description": [
-        "No updates to postgres, redis, php or python docker images and composer versions",
+        "No updates to postgres, redis, php or python docker images",
         "postgres: keep in sync with live",
         "redis: keep in sync with live",
-        "php: 8.2 upgrade needs to be managed manually",
-        "python: to be removed"
+        "php: 8.2 upgrade needs to be managed manually"
       ],
       "matchUpdateTypes": ["major", "minor", "patch"],
-      "matchManagers": ["dockerfile", "docker-compose", "composer"],
+      "matchManagers": ["dockerfile", "docker-compose"],
```

Comment:
Looking at this again, I'm not sure why you took composer out. The idea is to prevent php being updated in composer files, which could happen if you remove this manager.

Reply:
I got mixed up and thought it had been taken out of the other branch. Merge error, now fixed.